R/PCA_analysis.qmd

```diff
@@ -0,0 +1,58 @@
+---
+title: "Principle Component Analysis"
+format: html
+editor: visual
+---
+
+## Introduction
+
+Principal Component Analysis (PCA) is a dimensionality reduction technique used to reduce the number of features in a dataset while retaining most of the information.
+
+### Steps to Perform PCA in R
+
+-   We will load the `iris` data.
+-   Standardize the data and then compute PCA.
+
+```{r}
+suppressPackageStartupMessages({
+  library(plotly)
+  library(factoextra)
+})
+  
```

Comment:
```suggestion
```{r}
#| warning: true
  library(factoextra)
  
```

Reply:
You don't need the plotly library

Reply:
removed plotly

Reply:
@statasaurus is it fine now?